README.md

```diff
@@ -20,14 +20,16 @@
 
     ```
     az login
+    export KL_AZURE_SUBSCRIPTION_ID=<YOUR SUBSCRIPTION ID>
+    az account set --subscription $KL_AZURE_SUBSCRIPTION_ID
```

Comment:
This pretty much defeats the purpose of automating the setup (as much as possible). The original version set the environment variable in line 39 (deleted by this PR). Typically people do not know their subscription IDs and often do not know how to find them (especially if new to Azure). Therefore a better solution would be to given an instruction to ensure the right subscription is selected IFF they have more than one subscription (again most people do not). 

The goal is to make the README as simple to follow with no assumptions about the readers expertise.